bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitImpl.java

```diff
@@ -78,14 +74,14 @@
     public HomekitImpl(@Reference StorageService storageService, @Reference ItemRegistry itemRegistry,
             @Reference NetworkAddressService networkAddressService, Map<String, Object> config,
             @Reference MetadataRegistry metadataRegistry) throws IOException, InvalidAlgorithmParameterException {
-        this.storage = storageService.getStorage("homekit");
         this.networkAddressService = networkAddressService;
         this.settings = processConfig(config);
         this.changeListener = new HomekitChangeListener(itemRegistry, settings, metadataRegistry, storageService);
+        authInfo = new HomekitAuthInfoImpl(storageService.getStorage("homekit"), settings.pin);
```

Comment:
You should really use constants instead of literals when accessing storage.

Reply:
good point. change it also in HomekitAuthInfoImpl to constants.